.drone.yml

```diff
@@ -3,6 +3,7 @@ build:
   unit_tests:
     image: python:2-slim
     commands:
+      - rm -rf .tox
```

Comment:
okay for getting ci green, which to be noted this is not at yet, but not a desirable solution

Reply:
wondering if we should just go ahead and do what botocore does and separate the install to a different script, my fix for tox seems to be running afoul of tests that are dependent on the extant behavior.

Reply:
The same does not appear to work on travis.

Reply:
travis has its own config file.